### PR TITLE
Fix bugs for running phpmyadmin

### PR DESCRIPTION
**Pull request by iamxy**
1. Add ShowGrants to oldShow convert, phpmyadmin can run up now.
2. Fix a bug in Accept of FuncLocateExpr, handling the error while the Pos arg of LOCATE() not given.

### CODE REVIEW

File: ast/functions.go [hunk 1]
```diff
@@ -249,11 +249,13 @@ func (n *FuncLocateExpr) Accept(v Visitor) (Node, bool) {
 		return n, false
 	}
 	n.SubStr = node.(ExprNode)
-	node, ok = n.Pos.Accept(v)
-	if !ok {
-		return n, false
+	if n.Pos != nil {
```
Comment (qiuyesuifeng) [inline ast/functions.go]: We should add some tests to cover this.

File: executor/converter/convert_stmt.go [hunk 2]
```diff
@@ -822,6 +822,8 @@ func convertShow(converter *expressionConverter, v *ast.ShowStmt) (*stmts.ShowSt
 		oldShow.Target = stmt.ShowCreateTable
 	case ast.ShowDatabases:
 		oldShow.Target = stmt.ShowDatabases
+	case ast.ShowGrants:
```
Comment (qiuyesuifeng) [inline executor/converter/convert_stmt.go]: Test needed.

Comment (iamxy) [inline executor/converter/convert_stmt.go]: ok, i'll add some test cases for above

**Comment by qiuyesuifeng** [comment]
@shenli  PTAL

**Comment by iamxy** [comment]
@qiuyesuifeng @shenli PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**